packages/webdriver-utils/src/providers/automateProvider.js

```diff
@@ -83,12 +83,23 @@ export default class AutomateProvider extends GenericProvider {
           percyBuildUrl: this.buildInfo.url,
           state: 'begin'
         });
+        // Selenium Hub, set status error Code to 13 if an error is thrown
+        if (result?.status && result.status === 13) throw new Error(result?.value || 'Got invalid error response');
```

Comment:
```suggestion
        if (result?.status === 13) throw new Error(result?.value || 'Got invalid error response');
```

Reply:
would suggest removing the comment as well.

Reply:
Resolved